webknossos/tests/test_annotation.py

```diff
@@ -28,7 +28,7 @@ def test_annotation_from_zip_file() -> None:
         in annotation.metadata  # pylint: disable=unsupported-membership-test
     )
     assert len(list(annotation.get_volume_layer_names())) == 1
-    assert len(list(annotation.skeleton.flattened_graphs())) == 1
```

Comment:
Should we also test usage of deprecated identifiers?

Reply:
I don't think so.